JDK8/x64/Linux - Illegal Start of Expression in building

https://ci.adoptopenjdk.net/view/Failing%20Builds/job/build-scripts/job/jobs/job/jdk8u/job/jdk8u-linux-x64-openj9/854/console
```
12:22:38  Compiling tools to generate DDR pointer classes and structure stubs
12:22:38  /home/adoptopenjdk/workspace/build-scripts/jobs/jdk8u/jdk8u-linux-x64-openj9/workspace/build/src/openj9/debugtools/DDR_VM/src/com/ibm/j9ddr/StructureReader.java:219: error: illegal start of expression
12:22:38  		Map<String, String> aliasMap = loadAliasMap(() -> getAliasVersion());
12:22:38  		                                             ^
12:22:38  /home/adoptopenjdk/workspace/build-scripts/jobs/jdk8u/jdk8u-linux-x64-openj9/workspace/build/src/openj9/debugtools/DDR_VM/src/com/ibm/j9ddr/StructureReader.java:219: error: illegal start of expression
12:22:38  		Map<String, String> aliasMap = loadAliasMap(() -> getAliasVersion());
12:22:38  		                                                ^
12:22:38  /home/adoptopenjdk/workspace/build-scripts/jobs/jdk8u/jdk8u-linux-x64-openj9/workspace/build/src/openj9/debugtools/DDR_VM/src/com/ibm/j9ddr/StructureReader.java:229: error: illegal start of expression
12:22:38  		return loadAliasMap(() -> Long.toString(version));
12:22:38  		                     ^
12:22:38  /home/adoptopenjdk/workspace/build-scripts/jobs/jdk8u/jdk8u-linux-x64-openj9/workspace/build/src/openj9/debugtools/DDR_VM/src/com/ibm/j9ddr/StructureReader.java:229: error: illegal start of expression
12:22:38  		return loadAliasMap(() -> Long.toString(version));
12:22:38  		                        ^
12:22:39  4 errors
```
First instance of this occurring at Adopt.

--- a/debugtools/DDR_VM/src/com/ibm/j9ddr/StructureReader.java
+++ b/debugtools/DDR_VM/src/com/ibm/j9ddr/StructureReader.java
@@ -216,7 +216,7 @@ public class StructureReader {
 	}
 
 	private void applyAliases() throws IOException {
-		Map<String, String> aliasMap = loadAliasMap(() -> getAliasVersion());
+		Map<String, String> aliasMap = loadAliasMap(getAliasVersion());
 
 		for (StructureDescriptor thisStruct : structures.values()) {
 			for (FieldDescriptor thisField : thisStruct.fields) {
@@ -226,11 +226,11 @@ public class StructureReader {
 	}
 
 	public static Map<String, String> loadAliasMap(long version) throws IOException {
-		return loadAliasMap(() -> Long.toString(version));
+		return loadAliasMap(Long.toString(version));
 	}
 
-	private static Map<String, String> loadAliasMap(Supplier<String> versionSupplier) throws IOException {
-		String mapData = loadAliasMapData(versionSupplier);
+	private static Map<String, String> loadAliasMap(String version) throws IOException {
+		String mapData = loadAliasMapData(version);
 
 		mapData = stripComments(mapData);
 
@@ -268,8 +268,8 @@ public class StructureReader {
 		return version + variant;
 	}
 
-	private static String loadAliasMapData(Supplier<String> versionSupplier) throws IOException {
-		String streamAliasMapResource = "/com/ibm/j9ddr/StructureAliases" + versionSupplier.get() + ".dat";
+	private static String loadAliasMapData(String version) throws IOException {
+		String streamAliasMapResource = "/com/ibm/j9ddr/StructureAliases" + version + ".dat";
 		InputStream is = StructureReader.class.getResourceAsStream(streamAliasMapResource);
 
 		if (null == is) {

